attributes_test.go

```diff
@@ -342,3 +342,52 @@ func TestExtractError(t *testing.T) {
 	is.Len(attrs, 3)
 	is.EqualError(err, assert.AnError.Error())
 }
+
+func TestRemoveEmptyAttrs(t *testing.T) {
+	t.Parallel()
+	is := assert.New(t)
+
+	// do not remove anything
+	is.Equal(
+		[]slog.Attr{slog.Bool("bool", true), slog.Int("int", 42)},
+		RemoveEmptyAttrs(
+			[]slog.Attr{slog.Bool("bool", true), slog.Int("int", 42)},
+		),
+	)
+
```

Comment:
```suggestion
	is.Equal(
		[]slog.Attr{slog.Bool("bool", false), slog.Int("int", 42)},
		RemoveEmptyAttrs(
			[]slog.Attr{slog.Bool("bool", false), slog.Int("int", 42)},
		),
	)

```

It may be nice if there is a test for zero-values.

You can reject this because I have shallow experience for golang.

Reply:
I agree